web/src/lib/components/photos-page/actions/select-all-assets.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+  import CircleIconButton from '$lib/components/elements/buttons/circle-icon-button.svelte';
+  import SelectAll from 'svelte-material-icons/SelectAll.svelte';
+  import TimerSand from 'svelte-material-icons/TimerSand.svelte';
+  import {assetInteractionStore} from '$lib/stores/asset-interaction.store';
+  import {assetGridState, assetStore} from '$lib/stores/assets.store';
+  import {handleError} from '../../../utils/handle-error';
+  import {AssetGridState, BucketPosition} from "$lib/models/asset-grid-state";
+
+  let selecting = false;
+
+  const handleSelectAll = async () => {
+    try {
+      selecting = true;
+      let _assetGridState = new AssetGridState();
+      assetGridState.subscribe((state) => {
+        _assetGridState = state;
+      });
+
+      for (let i = 0; i < _assetGridState.buckets.length; i++) {
+        await assetStore.getAssetsByBucket(_assetGridState.buckets[i].bucketDate, BucketPosition.Unknown)
```

Comment:
Not 100% sure, but I think `BucketPosition.Unknown` will result in scrolling up issues

Reply:
@brighteyed  I am not sure either. Just selected it, because i needed to load all buckets.

Reply:
Just try to scroll up. Do you experience periodical shifts/jumps?

Reply:
@brighteyed nop scrolled multiple times now. Everything worked fine and as expected.

Reply:
Ah, intersection observer will eventually update position with correct value, so `Unknown` looks fine